spec/system/prior_authority/application_lists_spec.rb

```diff
@@ -18,17 +18,19 @@
     create(:prior_authority_application, laa_reference: 'LAA-DDDDD', status: 'draft', updated_at: 4.days.ago)
     create(:prior_authority_application, laa_reference: 'LAA-EEEEE', status: 'draft',
            office_code: 'OTHER', provider: create(:provider, :other))
-
-    visit prior_authority_applications_path
   end
 
-  it 'only shows right applications in the right tabs' do
-    within '#submitted' do
+  context 'only shows right applications in the right tabs' do
```

Comment:
Nit: Contexts should be about when a certain condition applies. Really I think you want to use a `describe` block, maybe something like `describe 'tab contents'`. Then your `it` lines should really read like a grammatical sentence, like `it 'has correct content in submitted tab'`.